### PR TITLE
Add benchmarks for octopus-pallets

### PR DESCRIPTION
**Pull request by anonymousGiga**
Add benchmarks for octopus-pallets

### CODE REVIEW

File: bridge/benchmarking/src/lib.rs [hunk 1]
```diff
@@ -1,304 +1,369 @@
...
+		assert_eq!(
+			name,
+			"empty".to_string().as_bytes().to_vec(),
+		);
+	}
+
+	force_unlock {
+		type Balance<T> = <<T as BridgeConfig>::Currency as Currency<<T as frame_system::Config>::AccountId>>::Balance;
+
+		let ass: Balance<T> = (1000_000_000_000_0000u128).checked_into().unwrap();
```
Comment (temajm) [inline bridge/benchmarking/src/lib.rs:260]:
```suggestion
		let ass: Balance<T> = (10_000_000_000_000_000u128).checked_into().unwrap();
```

File: bridge/benchmarking/src/lib.rs [hunk 2]
```diff
@@ -1,304 +1,369 @@
...
+		let amount: Balance<T> = (1000_000_000_000_000_000u128).checked_into().unwrap();
```
Comment (temajm) [inline bridge/benchmarking/src/lib.rs:261]:
```suggestion
		let amount: Balance<T> = (1_000_000_000_000_000_000u128).checked_into().unwrap();
```

File: bridge/benchmarking/src/lib.rs [hunk 3]
```diff
@@ -1,304 +1,369 @@
...
+	force_mint_nep141 {
+		let (caller, receiver) = create_default_asset::<T, pallet_assets::Instance1>(true);
+		<T as BridgeConfig>::Currency::make_free_balance_be(&caller, <T as BridgeConfig>::Currency::minimum_balance());
+		let asset_id = get_asset_id::<T>(vec![1,0,0,0]);
+	}: {
+		let ret = BridgePallet::<T>::force_mint_nep141(
+			RawOrigin::Root.into(),
+			asset_id,
+			receiver,
+			100000u32.into());
```
Comment (temajm) [inline bridge/benchmarking/src/lib.rs:293]:
```suggestion
			100_000u32.into());
```

File: bridge/benchmarking/src/lib.rs [hunk 4]
```diff
@@ -1,304 +1,369 @@
...
+		assert!(ret.is_ok());
+	}
+	verify {
+		assert_last_event::<T>(BridgeEvent::ForceNep141Minted {
+			asset_id,
+			who: caller,
+			amount: 100000u32.into(),
```
Comment (temajm) [inline bridge/benchmarking/src/lib.rs:300]:
```suggestion
			amount: 100_000u32.into(),
```

File: bridge/benchmarking/src/lib.rs [hunk 5]
```diff
@@ -1,304 +1,369 @@
...
+		.into());
+	}
+
+	set_token_price {
+		let caller: T::AccountId = whitelisted_caller();
+		let source = T::Lookup::unlookup(caller.clone());
+		let _ = BridgePallet::<T>::set_oracle_account(RawOrigin::Root.into(), source);
+		let origin = RawOrigin::Signed(caller.clone());
+	}: {
+		let ret = BridgePallet::<T>::set_token_price(origin.into(), 1000000);
```
Comment (temajm) [inline bridge/benchmarking/src/lib.rs:359]:
```suggestion
		let ret = BridgePallet::<T>::set_token_price(origin.into(), 1_000_000);
```

File: bridge/benchmarking/src/lib.rs [hunk 6]
```diff
@@ -1,304 +1,369 @@
...
+	}
+	verify {
+		assert_last_event::<T>(BridgeEvent::TokenPriceUpdated{
+			who: caller,
+			price: 1000000,
```
Comment (temajm) [inline bridge/benchmarking/src/lib.rs:364]:
```suggestion
			price: 1_000_000,
```

File: bridge/benchmarking/src/mock.rs [hunk 7]
```diff
@@ -1,419 +1,418 @@
...
+pub type Signature = MultiSignature;
+pub type Balance = u128;
+pub type Moment = u64;
+pub type Index = u64;
+pub type Hash = sp_core::H256;
+
+pub const MILLICENTS: Balance = 10_000_000_000;
+pub const CENTS: Balance = 1_000 * MILLICENTS;
+pub const DOLLARS: Balance = 100 * CENTS;
+pub const MILLISECS_PER_BLOCK: Moment = 3000;
```
Comment (temajm) [inline bridge/benchmarking/src/mock.rs:37]:
```suggestion
pub const MILLISECS_PER_BLOCK: Moment = 3_000;
```

File: bridge/benchmarking/src/mock.rs [hunk 8]
```diff
@@ -1,419 +1,418 @@
...
+pub const SECS_PER_BLOCK: Moment = MILLISECS_PER_BLOCK / 1000;
```
Comment (temajm) [inline bridge/benchmarking/src/mock.rs:38]:
```suggestion
pub const SECS_PER_BLOCK: Moment = MILLISECS_PER_BLOCK / 1_000;
```

File: bridge/benchmarking/src/mock.rs [hunk 9]
```diff
@@ -1,419 +1,418 @@
...
+pub const SLOT_DURATION: Moment = MILLISECS_PER_BLOCK;
+pub const EPOCH_DURATION_IN_BLOCKS: BlockNumber = 1 * MINUTES;
+pub const MINUTES: BlockNumber = 60 / (SECS_PER_BLOCK as BlockNumber);
+
+parameter_types! {
+	pub const BlockHashCount: BlockNumber = 2400;
```
Comment (temajm) [inline bridge/benchmarking/src/mock.rs:44]:
```suggestion
	pub const BlockHashCount: BlockNumber = 2_400;
```

**Review by temajm — COMMENTED** [review]
I think it's more readable.

**Review by en — APPROVED** [review]

**Pull request merged**